src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -158,11 +162,16 @@ public class UTIL_ACCT_Naming {
         accountNamingFormat = checkAccountFormat();
 
         NameSpec ns;
+
+        List<Contact> returnSortedContacts = new List<Contact>();
+		returnSortedContacts = sortContacts(cons);
```

Comment:
It looks like these two lines can just be truncated down to: `List<Contact> returnSortedContacts = sortContacts(cons);`